ERRATA.md

```diff
@@ -80,3 +80,27 @@ This document includes errata for the [Activity Streams](https://www.w3.org/TR/a
   - Unlike `latitude` and `longitude`, the domain of the `altitude` term is the `Object` type. The `altitude` term should be included in the list of properties of an `Object`. Because `altitude` is primarily documented as a property of a `Place`, publishers should not include `altitude` on objects that are not of type `Place`, and consumers should accept objects with this property that aren't of type `Place`.
 
   - The domain of the `attributedTo` property is both `Link` and `Object`. `attributedTo` should be included in the list of properties of a `Link`.
+
+  - Example 79 has  an `Image` object with `width` and `height` properties, which
+    are only allowed on `Link` objects. One alternative is to use a `Link` object
```

Comment:
I dont believe there is any normative text anywhere in the as2 TR that justifies a conclusion that these properties "are only allowed on Link objects".
```suggestion
    are defined only on `Link` objects. One alternative is to use a `Link` object
```

I think a more appropriate resolution to the confusion here would be to add `Image` or `Object` to https://github.com/w3c/activitystreams/blob/main/vocabulary/index.html#L5531